server-python/app.py

```diff
@@ -0,0 +1,90 @@
+from flask import (
```

Comment:
If there's a copyright license header required here LMK

Reply:
Let's use an Apache 2 copyright, e.g. https://github.com/google-gemini/generative-ai-python/blob/main/samples/chat.py#L2-L14

Reply:
Done. I've added it now.

Reply:
Thank you!